modules/mocked_authentication/spec/lib/mocked_credential/service_spec.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'mocked_credential/service'
+
+describe MockedAuthentication::MockedCredential::Service do
+  let(:code) { '6805c923-9f37-4b47-a5c9-214391ddffd5' }
+  let(:token) do
+    {
+      access_token: 'AmCGxDQzUAr5rPZ4NgFvUQ',
+      token_type: 'Bearer',
+      expires_in: 900,
+      id_token: 'eyJraWQiOiJmNWNlMTIzOWUzOWQzZGE4MzZmOTYzYmNjZDg1Zjg1ZDU3ZDQzMzVjZmRjNmExNzAzOWYLOL' \
+                'QzNjFhMThiMTNjIiwiYWxnIjoiUlMyNTYifQ.eyJzdWIiOiI2NWY5ZjNiNS01NDQ5LTQ3YTYtYjI3Mi05Z' \
+                'DYwMTllN2MyZTMiLCJpc3MiOiJodHRwczovL2lkcC5pbnQuaWRlbnRpdHlzYW5kYm94Lmdvdi8iLCJlbWF' \
+                'pbCPzPmpvaG4uYnJhbWxleUBhZGhvY3RlYW0udXMiLCJlbHqZbF92ZXJpZmllZCI6dHJ1ZSwiZ2l2ZW5fb' \
+                'mFtZSI6IkpvaG4iLCJmYW1pb1bRfbmFtZSI6IkJyYW1sZXkiLCJiaXJ0aGRhdGUiOiIxOTg5LTAzLTI4Ii' \
+                'wic29jaWFsX3NlY3VyaXR5X251bWJlciI6IjA1Ni03Ni03MTQ5IiwidmVyaWZpZWRfYXQiOjE2MzU0NjUy' \
+                'ODYsImFjciI6Imh0dHA6Ly9pZG1hbmFnZW1lbnQuZ292L25zL2Fzc3VyYW5jZS9pYWwvMiIsIm5vbmNlIj' \
+                'oiYjIwNjc1ZjZjYmYwYWQ5M2YyNGEwMzE3YWU3Njk5OTQiLCJhdWQiOiJ1cm46Z292OmdzYTpvcGVuaWRj' \
+                'b25uZWN0LnByb2ZpbGVzOnNwOnNzbzp2YTpkZXZfc2lnbmluIiwianRpIjoicjA1aWJSenNXSjVrRnloM1' \
+                'ZuVlYtZyIsImF0X2hhc2giOiJsX0dnQmxPc2dkd0tKemc2SEFDYlJBIiwiY19oYXNoIjoiY1otX2F3OERj' \
+                'SUJGTEVpTE9QZVNFUSIsImV4cCI6MTY0NTY0MTY0NSwiaWF0IjoxNjQ1NjQwNzQ1LCJuYmYiOjE2NDU2ND' \
+                'A3NDV9.S3-8X9clNcwlH2RU5sNoYf9HXpcgVK9UGUJumhL2-3rvznrt6yGvkXvY4FuUzWEcI22muxUjbbs' \
+                'ZHjCfDImZ869NTWsI-DKohSNmNnyOom29LJRymJTn3htI5MNmpGwbmNWNuK5HgerPZblL44N1a_rqfTF4l' \
+                'ANQX0u52iIVDarcexpX0e9yS1rEPqi3PDdcwN_1tUYox4us9rgzRZaaoa4iTlFfovY7dfgo_ewqv2EDh7J' \
+                'SfJJQhFhyabkJ9HgNkkc4m0SHqztterZ6lHgIoiJdQot6wsL9pQTYzFzgHV830ltpjVUcLG5vMXw4Kqs3B' \
+                'N9tdSToHdB50Paxyfq9kg'
+    }
+  end
+  let(:user_info) do
+    OpenStruct.new({
+                     sub: user_uuid,
+                     iss: 'https://idp.int.identitysandbox.gov/',
+                     email: email,
+                     email_verified: true,
+                     given_name: first_name,
+                     family_name: last_name,
+                     address: address,
+                     birthdate: birth_date,
+                     social_security_number: ssn,
+                     verified_at: 1_635_465_286
+                   })
+  end
+  let(:first_name) { 'Bob' }
```

Comment:
These values were specifically set for the `logingov/service` spec, and they had to be set to these values to match the values from the saved token from a real login.gov `/userinfo` response. Since we're just mocking here, I'd say we should just make all these values arbitrary and simplify them. My personal preference is this style:
`let(:first_name) { 'some-first-name' }`
`let(:birth_date) { 'some-birth-date' }`
...
etc
To me at least, this lets the person reading the specs know that, technically, the value of each of these doesn't truly matter, as long as it matches the expected format of the output